.ci/ci_functions.sh

```diff
@@ -0,0 +1,46 @@
+function configure () {
```

Comment:
Please add some comments what is this function for. Check also other functions in the file

Reply:
Done